list.html

```diff
@@ -2,69 +2,75 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>Book list</title>
+  <title>Book List</title>
   <meta name="description" content="training, book list card">
   <meta name="author" content="Wolox">
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
-<body>
-  <div class="container list">
-    <div class="content">
-      <div class="list__card">
-        <div class="list__cover">
-          <img class="list__cover__image" src="assets/book-cover.png" alt="Wolox book cover">
+<body class="body">
+  <nav class="nav-bar">
+    <a class="logo" href="#home"></a>
```

Comment:
Page logo is not decorative element. Please add it as <image />

Reply:
Ok